generators/server/jdl/application-definition.ts

```diff
@@ -53,7 +55,7 @@ const applicationConfig: JDLApplicationConfig = {
     jdlOptions.map(option => [
       option.name,
       {
-        type: messageBrokerOption.type,
+        type: option.type,
```

Comment:
This change doesn't seem related to the feature.

Reply:
Yes it's a fix.
I can split non related changes if needed.

Reply:
As long as it's intentional, I'm OK with it.